xml_inscriptions/transcribed/NJ.Princ.PU.AM.G.Pc132-I301.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="utf-8"?>
+<?oxygen RNGSchema="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" type="xml"?><TEI xmlns:xi="http://www.w3.org/2001/XInclude" xmlns="http://www.tei-c.org/ns/1.0">
+   <teiHeader><!-- testing1 -->
+      <fileDesc>
+         <titleStmt>
+            <title>NJ.Princ.PU.AM.G.Pc132-I301</title>
+         </titleStmt>
+         <publicationStmt>
+            <xi:include  href="http://library.brown.edu/usep_data/resources/include_publicationStmt.xml">
+               <xi:fallback>
+                  <p>The Publications Statement should appear here. (c) Brown University</p>
+               </xi:fallback>
+            </xi:include>
+            <idno type="USEpigraphy" xml:id="NJ.Princ.PU.AM.G.Pc132-I301">NJ.Princ.PU.AM.G.Pc132-I301</idno>
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc>
+               <msIdentifier>
+                  <region>NJ</region>
+                  <settlement>Princ</settlement>
+                  <institution>PU</institution>
+                  <repository>AM</repository>
+                  <idno>Pc132-I301</idno>
+               </msIdentifier>
+               <msContents>
+                  <textLang mainLang="grc" otherLangs=""/>
+                  <msItem>
+                     <p>Fragment of a Greek inscription</p>
+                  </msItem>
+               </msContents>
+               <physDesc>
+                  <!--  *************************************  <objectDesc>  *********************************  -->
+                  
+                  <objectDesc ana="#fragment">
+                     <supportDesc ana="#stone.marble">
+                        <support>
+                           <p>white-grey veined marble fragment</p>
+                           <dimensions type="surface" unit="cm">
+                              <height>12.6</height>
+                              <width/>
```

Comment:
Do we not have width information, or did this just get left out by accident?

Reply:
We don't have the width information.